app/models/user/User.scala

```diff
@@ -127,6 +127,52 @@ class UserDAO @Inject()(sqlClient: SqlClient)(implicit ec: ExecutionContext)
       parsed <- parseAll(r)
     } yield parsed
 
+  def buildSelectionPredicates(isEditableOpt: Option[Boolean],
+                               isTeamManagerOrAdminOpt: Option[Boolean],
+                               isAdminOpt: Option[Boolean],
+                               requestingUser: User)(implicit ctx: DBAccessContext): Fox[SqlToken] =
+    for {
+      accessQuery <- accessQueryFromAccessQ(listAccessQ)
+      editablePredicate = isEditableOpt match {
+        case Some(isEditable) =>
+          val usersInTeamsManagedByRequestingUser =
+            q"(SELECT _user FROM webknossos.user_team_roles WHERE _team IN (SELECT _team FROM webknossos.user_team_roles WHERE _user = ${requestingUser._id}  AND isteammanager=TRUE)))"
```

Comment:
```suggestion
            q"(SELECT _user FROM webknossos.user_team_roles WHERE _team IN (SELECT _team FROM webknossos.user_team_roles WHERE _user = ${requestingUser._id} AND isTeamManager)))"
```

Reply:
(I think the `=TRUE` can be omitted).

Let’s stick with the capitalization used in schema.sql (e.g. isTeamManager, isAdmin). While postgres queries are case insensitive, I’d say that yields the best readability for us.